shortfin/shortfin/llm/impl/service_v1_cli.py

```diff
@@ -0,0 +1,91 @@
+
+import asyncio
+import argparse
+import numpy
+import sys
+
+from transformers import LlamaTokenizer
+
+from iree.runtime import (  # type: ignore
+    HalElementType,
+)
+
+from shortfin.framework.session import DeviceSession
+
+from shortfin.llm.attn_block_cache import (
+    create_attn_block_cache_module,
+    AttnBlockCache,
+)
+
+from shortfin.llm.config import (
+    CacheParams,
+    ModelParams,
+    ServiceParams,
+)
+
+from shortfin.llm.impl.service_v1 import GenerateServiceV1
+from shortfin.llm.service import GenerateRequest
+
+def setup(vmfb_path, config_path, gguf_path):
+    from iree.runtime._binding import disable_leak_checker  # type: ignore
+    model_params = ModelParams.load_json(config_path)
+
+    cache_params = CacheParams(
+        model=model_params, device_block_count=128, block_pos_stride=16)
+
+    disable_leak_checker()
```

Comment:
👀  leaks?

Reply:
Unfortunately yes - I think a decent chunk of this is how python releases resources. There may be a better way to track this down but its replicated on the existing service work.